packages/integration-karma/test/api/CustomElementConstructor-getter/index.spec.js

```diff
@@ -22,6 +15,13 @@ it('should throw when trying to claim abstract LightningElement as custom elemen
 });
 
 if (SUPPORTS_CUSTOM_ELEMENTS) {
+    it('should return a custom element', () => {
+        class Test extends LightningElement {}
+        const TestCustomElement = Test.CustomElementConstructor;
+
+        expect(typeof TestCustomElement).toBe('function');
+    });
```

Comment:
Could not figure out why, but this started throwing in IE. Didn't seem worth investigating beyond that.

Reply:
Aha, so you're saying that the fact that the behavior changed here is not a big deal? What if someone is using their own custom elements polyfill with `CustomElementConstructor` in IE11 though?